File: Programmers/Lv0/조건에맞게수열변환하기1.py
def solution(arr):
    for idx in range(len(arr)):
        num = arr[idx]
        if num >= 50 and num % 2 == 0:
            num = num // 2
        elif num < 50 and num % 2 == 1:
            num = num * 2
        arr[idx] = num
    return arr

File: Programmers/Lv0/조건에맞게수열변환하기1.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr) {
    for (int &num : arr)
    {
        if (num >= 50 && num % 2 == 0)
            num /= 2;
        else if (num < 50 && num % 2 == 1)
            num *= 2;
    }
    return arr;
}